src/components/Feedback/_feedback.scss

```diff
@@ -0,0 +1,38 @@
+@include exports('addons-bmrg-feedback') {
+  .#{$prefix}--bmrg-feedback-container {
+    .#{$prefix}--modal-container {
+      height: rem(448px);
+    }
+
+    .#{$prefix}--bmrg-feedback {
+      color: $bmrg-brand-white;
+      height: rem(100px);
+      margin-top: rem(16px);
+      width: 100%;
+
+      a {
```

Comment:
![image](https://user-images.githubusercontent.com/4127669/151838741-f4e5da08-f73e-4ff1-8d97-72f819e0f95b.png)

We should use another theme token here for a11y and making sure it works for both themes. For the boomerang theme the `bmrg-tertiary-color` is the same as the background color.